AppointmentsMicroService/BusinessLogicLayer/Services/IAppointmentService.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using DataAccessLayer.Entityes;
+
+namespace BusinessLogicLayer.Services
+{
+    /// <summary>
+    /// Abstrarion of Service for Appointments.
+    /// </summary>
+    public interface IAppointmentService
```

Comment:
I understand the reason why you decided to have one controller and service. But two will look cleaner and smaller.